### PR TITLE
[AUD-1597] Add AboutScreen

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

- Adds AboutScreen
- Adds Link core component to make generating links easier
- Adds Text core component that hooks into existing typography

image

### CODE REVIEW

File: packages/mobile/src/components/core/Text.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Text as TextBase, TextProps as TextBaseProps } from 'react-native'
```
Comment (sliptype) [inline packages/mobile/src/components/core/Text.tsx:1]:
```suggestion
import { Text as RNText, TextProps as RNTextProps } from 'react-native'
```

Just to stay consistent with other places

File: packages/mobile/src/components/core/Text.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  ({ typography, palette }, { variant, noGutter }) => ({
+    root: {
+      ...typography[variant],
+      color: palette.neutral,
+      ...(noGutter && { marginBottom: 0 })
+    }
+  })
+)
+
+export const Text = (props: TextProps) => {
```
Comment (sliptype) [inline packages/mobile/src/components/core/Text.tsx:20]: Should we add a `@deprecated` jsdoc comment to our other `Text` component?

File: packages/mobile/src/screens/settings-screen/SettingRowLabel.tsx [hunk 3]
```diff
@@ -1,24 +1,60 @@
...
+      icon: ComponentType<SvgProps>
+    })
+
 export const SettingsRowLabel = (props: SettingsRowLabelProps) => {
-  const { label, iconSource } = props
+  const { label } = props
   const styles = useStyles()
+
+  const renderIcon = () => {
+    if ('iconSource' in props) {
```
Comment (sliptype) [inline packages/mobile/src/screens/settings-screen/SettingRowLabel.tsx:37]: Neat 📸

**Review by sliptype — APPROVED** [review]
Fire

File: packages/mobile/src/components/core/Text.tsx (hunk 2 above)
Comment (dylanjeffers) [reply packages/mobile/src/components/core/Text.tsx:20]: down at some point, didnt want to make too much of an assumption there, was hoping to try out this other text component for a sec to see how it goes? will add a note though.

**Pull request merged**